selenium_tests/account_admin/account_admin_base_test_case.py

```diff
@@ -0,0 +1,36 @@
+from urlparse import urljoin
+
+from django.conf import settings
+
+from selenium_tests.base_selenium_test_case import BaseSeleniumTestCase
+from selenium_tests.pin.page_objects.pin_login_page_object import PinLoginPageObject
+from selenium_tests.account_admin.page_objects.account_admin_dashboard_page_object import AccountAdminDashboardPage
+
+
+class AccountAdminBaseTestCase(BaseSeleniumTestCase):
+    """
+    Bulk Create base test case, all other tests will subclass this class
+    """
+    @classmethod
+    def setUpClass(cls):
+        """
+        setup values for the tests
+        """
+        super(AccountAdminBaseTestCase, cls).setUpClass()
+
+        driver = cls.driver
+        cls.USERNAME = settings.SELENIUM_CONFIG.get('selenium_username')
+        cls.PASSWORD = settings.SELENIUM_CONFIG.get('selenium_password')
+        cls.CANVAS_BASE_URL = settings.SELENIUM_CONFIG.get('canvas_base_url')
+        cls.TOOL_RELATIVE_URL = settings.SELENIUM_CONFIG.get('course_info_tool_relative_url')
```

Comment:
This should be its own selenium_config setting, given the test suite for this page is independent of the test suite for the course_info tool.